Content.Server/Psionics/PotentialPsionicComponent.cs

```diff
@@ -0,0 +1,21 @@
+namespace Content.Server.Psionics
+{
+    [RegisterComponent]
+    public sealed partial class PotentialPsionicComponent : Component
+    {
+        /// <summary>
+        /// The base chance of an entity rolling psychic powers, which is increased by other modifiers such as glimmer.
+        /// </summary>
+        /// <remarks>
+        /// I have increased this to 10% up from its original value of 2%, because I estimate that most people won't take the Latent Psychic trait
```

Comment:
Not sure if this should be increased; :shrug: I appreciate the reasoning but this could backfire. Maybe 5%?

Reply:
I'm quite intentionally keeping it at 10% for the baseline, it's to make sure that people who take the trait can much more consistently interact with the system, makes it more flashy to new players. Yea, that also means it's way more common. Other servers can and should reduce it on their end if desired, but for the default we want people to interact with psionics.